api/handlers/api.go

```diff
@@ -30,7 +30,8 @@ func (a *App) New() *mux.Router {
 
 	u := User{DB: databases.NewUserDatabase(a.dbHelper)}
 	c := Community{DB: databases.NewCommunityDatabase(a.dbHelper)}
-	n := search.Name{DB: databases.NewCivilianDatabase(a.dbHelper)}
+	n := search.Name{DB: databases.NewCivilianDatabase(a.dbHelper), VerifyInCommunity: u.VerifyInCommunity()}
```

Comment:
suggestion: for now we don't need to verify in the community

```suggestion
	n := search.Name{DB: databases.NewCivilianDatabase(a.dbHelper)}
```